Stripe/StripeiOSTests/CustomerAdapterTests.swift

```diff
@@ -123,7 +119,6 @@ class CustomerAdapterTests: APIStubbedTestCase {
 
         XCTAssertEqual(pms.count, 1)
         XCTAssertEqual(pms[0].stripeId, expectedPaymentMethods[0].stripeId)
-        await waitForExpectations(timeout: 2)
```

Comment:
These were giving warnings:
```
Instance method 'waitForExpectations' is unavailable from asynchronous contexts; Use await fulfillment(of:timeout:enforceOrder:) instead; this is an error in Swift 6
```

and didn't seem necessary since we already wait for `fetchPaymentMethods` to return, so I removed it.